Update: Alerts & Notifications (#2306)
getsentry/sentry-docs: src/docs/product/alerts-notifications/index.mdx

description: "Sentry categorizes notifications into alerts and other types of notifications. Learn more about how these are managed."
---

> Alerts and notifications are customizable to your organization's needs with options to pair with [integrations](/product/integrations/), flexible configurations, and choices for both [error](#issue-alerts) or [performance](#metric-alerts) alerting. Conversely, you can keep alerts and notifications as they are with Sentry's default settings. Alerts and notifications show the impact of errors on your users, help uncover the error type, and monitor your application's performance.

Reviewer: much better!

## Alerting for Errors & Performance

Alerts notify you about problems in your application and can connect with many supported Sentry [integrations](/product/integrations/). Set alerts on both issues and performance metrics.

To change the frequency of your alerts, configure your [Alert Subscription](/product/alerts-notifications/alerts/#alert-subscription).

### Issue Alerts

> [Issue alerting](/product/alerts-notifications/alerts/#issue-alerts) sets alerts on changes within grouped error events; also known as, issues. Create alerts for issues such as:

Reviewer: ```suggestion
[Issue alerting](/product/alerts-notifications/alerts/#issue-alerts) sets alerts on changes within grouped error events, or _issues_. Create alerts for issues such as:
```

- Issues affecting many users
- [Event](/platforms/javascript/enriching-events/) frequency
- Reoccuring events with specific [tags](/platforms/javascript/enriching-events/tags/)

### Metric Alerts

> [Metric alerting](/product/alerts-notifications/alerts/#metric-alerts) monitors macro-level [metrics](/product/performance/metrics/) for both error events and transaction events. A metric alert is triggered when a [threshold](/product/performance/metrics/#p50-threshold) is breached. You can set the threshold for metrics using the value of an aggregate function like `count()` or `avg()` to see how errors are impacting your user's experience, the [performance](/product/performance/) speed of your application, etc.

Reviewer: Avoid Latin abbreviations.

> Sentry categorizes notifications broadly into alerts and non-alert notifications.

Reviewer: Can we give a little more info here about the impact of these alerts and notifications? Because all this tells the user is what Sentry does (useful), but we want to explain better what the impact is of their choices here, right? We get a lot of comments that Sentry is too noisy, so here's where users can understand why and how to set things up so communications are useful and valid for their org.

## Alerts
## Error Alerting

> [Alerts](/product/alerts-notifications/alerts/) notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.

Reviewer: This is a bit confusing with the new heading because we discuss alerting on both errors and performance, but the title is errors...?

> Alerts notify you about problems in your application and can be sent to many supported integrations. You can set alerts on both errors and performance data.

Reviewer: Can we be a bit more explicit here? Perhaps "Alerts let you know of problems occurring in your application, and can be sent via email, chatops, and many other channels depending on the integrations you've created using our [Integration Platform](link). 

The last sentence here is confusing given that we're in as section entitled "Error Alerts"

### Issue Alerts

> Sentry automatically groups individual error events into issues based on stack trace and other factors. You can then set alerts on changes in those issues (such as new issues or issues affecting many users), or on the raw events (such as frequency of events with a given tag).

Reviewer: Do we need to explain grouping here? It seems like an implementation detail that's less important than our topic, which is alerting.

"you can then set alerts..." this makes it seem like it's a second step, but I don't think it is. Perhaps more clear as "Issue alerts are triggered when you need them, such as when a new issue arises or affects a threshold of users or frequency."

For more details, see the [full documentation on Issue Alerts](/product/alerts-notifications/alerts/#issue-alerts).

### Metric Alerts

> Use metric alerts on error data to compute the number of events or users affected by events in your project. A metric is the value of an aggregate function like `count()` or `avg()`. You can filter events by attributes and tags, which is particularly useful for aggregating across events that aren't grouped into single issues.

Reviewer: Perhaps start with clear definition of metric alerts: "Metric alerts are an aggregate of a function, such as `count()` or `avg()`. They function in a distinctly different fashion from issue alerts, which focus on individual issues." Might also be good to provide an example of each on this intro page.

For more details, see the [full documentation on Metric Alerts](/product/alerts-notifications/alerts/#metric-alerts).

## Performance Alerting

> Performance empowers you to see everything from macro-level [metrics](/product/performance/metrics/) to micro-level [spans](product/performance/event-detail/).

Reviewer: yes, but how does this impact alerts :) - keep the focus tightly on the topic.

### Metric Alerts

> Create alerts for performance metrics such as:

Reviewer: this is great. It's clear, specific, and concise (also bulleted lists increase readability)

> Create alerts for performance metrics such as:

Reviewer: Add a definition before you tell folks what they can create this for.
